include/experimental/__p1673_bits/blas1_vector_idx_abs_max.hpp

```diff
@@ -48,24 +48,23 @@ namespace experimental {
 inline namespace __p1673_version_0 {
 namespace linalg {
 
-// FIXME (Hoemmen 2021/05/28) Latest version of P0009 (mdspan) uses size_t
-// instead of ptrdiff_t, but the implementation hasn't changed yet.
 template<class ElementType,
-         ptrdiff_t ext0,
+         extents<>::size_type ext0,
          class Layout,
          class Accessor>
-ptrdiff_t vector_idx_abs_max(
+extents<>::size_type vector_idx_abs_max(
   std::experimental::basic_mdspan<ElementType, std::experimental::extents<ext0>, Layout, Accessor> v)
 {
   using std::abs;
+  using size_type = typename extents<>::size_type;
   using magnitude_type = decltype(abs(v(0)));
 
   if (v.extent(0) == 0) {
     return -1;
```

Comment:
Uh oh, we're returning -1 here.  This is actually a bug in the proposal itself (it says it returns -1 in this case).  I'll add an issue to fix the proposal.  We can come back around and fix this code later.

Reply:
Also, this function is called `idx_abs_max` in the proposal.  I'll file a separate issue for that too.